docs/source/introduction/makeyourown.rst

```diff
@@ -244,6 +250,45 @@ The file normalize.py contains the actual normalizer: Its only job is to parse r
                 yield ft
                 yield f
 
+Yielding a DeclaredScanProfile
+==============================
+
+Additionally, normalizers can yield DeclaredScanProfiles.
+This is useful if you want to avoid the manual work of raising these levels through the interface for new objects.
+Perhaps you have a trusted source of IP addresses and hostnames you are allowed to scan intrusively.
+Or perhaps you do not have intrusive boefjes enabled and always want to perform a DNS scan on new IP addresses.
+
+As an example, see these lines in `kat_external_db/normalize.py`:
+
+.. code-block:: python
+
+  def run(normalizer_meta: NormalizerMeta, raw: Union[bytes, str]) -> Iterator[OOI]:
+   ...
+   yield ip_address
+   yield DeclaredScanProfile(reference=ip_address.reference, level=3)
+
+
+This indicates that the ip_address should automatically be assigned a declared scan profile of level 3.
+
+The safeguard here is that you `always` need to specify which normalizers are allowed to add these scan profiles.
+This must be done through the ``SCAN_PROFILE_WHITELIST`` environment variable, which is a json-encoded mapping
+of normalizer ids to the `maximum` scan level they are allowed to set.
+An example value would be ``SCAN_PROFILE_WHITELIST='{"kat_external_db_normalize": 2}'``.
+
+When a higher level has been yielded, it will be lowered to the maximum.
+Combining the code and whitelist above would therefore be equivalent to combining this code with the whitelist:
+
+.. code-block:: python
+
+  def run(normalizer_meta: NormalizerMeta, raw: Union[bytes, str]) -> Iterator[OOI]:
+   ...
+   yield ip_address
+   yield DeclaredScanProfile(reference=ip_address.reference, level=2)
+
+
+Of course, when the normalizer id is not present in the whitelist, the yielded DeclaredScanProfiles are ignored.
```

Comment:
```suggestion
Of course, when the normalizer id is not present in the whitelist, the yielded `DeclaredScanProfiles` are ignored.
```

Reply:
Not sure if the single quote is what we want, but I did some updates (see updated screenshot also)